cheroot/server.py

```diff
@@ -1155,6 +1156,28 @@ def close(self):
             # Apache does, but not today.
             pass
 
+    def get_peer_uid(self):
```

Comment:
I'd prefer a method returning the whole tuple of creds and 3 `@property` methods returning each of 'em.